packages/cli/src/utils/analytics.js

```diff
@@ -21,13 +21,36 @@ const shouldSkipAnalytics = (mode) =>
   process.env.DISABLE_ZAPIER_ANALYTICS ||
   mode === ANALYTICS_MODES.disabled;
 
+const cleanValues = (args) => {
+  // List of arguments we don't want to record.
+  const doNotRecord = [
+    'key-value pairs...',
+    'keys...',
+    'path',
+    'email',
+    'message',
+    'user',
+    'account',
+    'redirect-uri',
+    'inputData',
+  ];
```

Comment:
It'd be easy to forget to update this list when developing a new CLI command or adding a command argument.

Or, do we need to capture the _values_ of arguments and flags? I think we can learn enough by capture only the _keys_ of arguments and flags.

Reply:
I was thinking the extra detail enables more precise segmentation of user behavior but we don't need to be that granular right now! I'll update this!

If we want this in the future, do you think having the `doNotRecord` list as an exported asset in `ZapierBaseCommand.js` close to where new CLI arguments/commands are added would be a sufficient safeguard?

Reply:
Thanks for updating!

Regarding:

> do you think having the `doNotRecord` list as an exported asset in `ZapierBaseCommand.js` close to where new CLI arguments/commands are added would be a sufficient safeguard?

I still don't think that would be sufficient. I think an allowlist (like `doRecord`) instead of a denylist would be a better safeguard. So, even if we forget to update the allowlist, we won't accidentally send unwanted data to the analytics endpoint.